src/tests/__snapshots__/AddRecipe.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders correctly 1`] = `null`;
```

Comment:
Where is this generated from?

Reply:
This is generated in AddRecipe.test.tsx from lines 50 to 60. But it is clearly not working correctly. I will look into it